descartes_light/solvers/include/descartes_light/solvers/bgl/impl/bgl_solver.hpp

```diff
@@ -77,7 +77,7 @@ std::vector<VertexDesc<FloatType>> BGLSolverBase<FloatType>::reconstructPath(con
   VertexDesc<FloatType> v = target;
   path.push_back(v);
 
-  for (VertexDesc<FloatType> u = predecessor_map_.at(v); u != v; v = u, u = predecessor_map_.at(v))
+  for (VertexDesc<FloatType> u = predecessor_map_[v]; u != v; v = u, u = predecessor_map_[v])
```

Comment:
I would keep `at` since it throws an exception on error rather than producing a seg-fault